.github/workflows/ci.yaml

```diff
@@ -201,6 +211,11 @@ jobs:
           CUDA_VER: "11.6.0"
           CENTOS_VER: "8"
 
+        - DOCKERIMAGE: linux-anvil-aarch64-cuda
+          DOCKERTAG: "11.7"
+          CUDA_VER: "11.7.0"
+          CENTOS_VER: "8"
```

Comment:
We may need to change this to use some other image (probably UBI) as CentOS 8 has been dropped